### ISSUE TITLE
npm start should start

### ISSUE THREAD

**Issue by hekike** [emartech/boar-stack-boilerplate#1]

**Comment by Valetudox** [comment]
You are right. Right now npm start only works when you run in the /dist folder. 

Maybe, we should change the npm start script to run gulp start. And in the build process we should replace that to the original one.

What do you think?

**Comment by hekike** [comment]
Yes it's a good a idea. Let's do that!
